util/reggen/reg_top.sv.tpl

```diff
@@ -241,8 +256,21 @@ module ${mod_name} (
   );
 
   logic err_q;
+<%
+  clk_lc_found = False;
+  for clock in rb.clocks.values():
+    if clock.clock == "clk_lc_i":
+      clk_lc_found = True;
+    endif
+  endfor
+%>
```

Comment:
Hi @jeoongp , it seems you're change somehow introduces additional line breaks here to the auto-generated `block_reg_top.sv` files of many blocks which shouldn't be touched by this PR at all. At the moment, the PR thus changes 75 files. Can you please try to remove these additional line breaks? It will ease PR the review a lot.

Reply:
No problem. I removed comments in reg_top.sv.tpl and the number of changed files is reduced to 29.